### DISCUSSION TITLE
Click action on Image

### DISCUSSION THREAD

**Discussion by jaz000** [Budibase/budibase#7186] [Help] [answered]
Is there a way I can add a click action to an image - to navigate to another screen (with  parameters?)

Alternatively, is there a way I can create an invisible button with an image (the only visible part).

Or some clever way I can combine the two ...

thanks!

**Comment by R2bEEaton** [comment, marked as answer]
Here's a sort of hacky way to do this, since Budibase doesn't support it natively (I think it totally should).

You can create a card with an image and set the click action, which allows you to click on the whole thing. Then use an embed with some CSS to get rid of the rest of the card, leaving just the image.

Here's an export if you want to examine this further.
[Test-export-1660074651285.txt](https://github.com/Budibase/budibase/files/9294133/Test-export-1660074651285.txt)

**Reply by jaz000** [reply]
Wow that's very clever, and basically works.

The only problem is that I can't get it the right size for my images, I have white space below (or to the right if I set horizontal card).
Seems to have a minimum width of ~270px. Some of my images are landscape and some portrait ...

**Reply by melohagan** [reply]
I had been working on clickable containers which would also allow for this use case. Hopefully I can get that sorted. https://github.com/Budibase/budibase/pull/5829

**Reply by R2bEEaton** [reply]
> Wow that's very clever, and basically works.
> 
> The only problem is that I can't get it the right size for my images, I have white space below (or to the right if I set horizontal card). Seems to have a minimum width of ~270px. Some of my images are landscape and some portrait ...

I'd say the solution to this would be more CSS which you can experiment with using a browser's "Inspect" feature.

@melohagan yeah I'd totally want to see something like #5829. I like that you automatically disable buttons inside a clickable container. I feel like the card solution is a good substitute for what you want to add, but it needs more options that would be a pain to add individually - which your container solution solves.
